katago_server/trainings/tasks/update_bayesian_rating.py

```diff
@@ -13,6 +13,8 @@ def update_bayesian_rating():
     :return:
     """
     current_run = Run.objects.select_current()
+    if current_run is None:
+        return
```

Comment:
We should throw a custom error so sentry (the bug catching platform) is notified.

Reply:
Is it a bug to have no run? Or do we simply just not want to update ratings if we're in the middle? Say we deactivate a run, but it's a few days until we begin the next run.

Reply:
The proper way would be that `update_bayesian_rating` takes some params, for instance the run name.

Thus we can have two run in parallel and thus two task for update each run.

Line 16 would become:

```python
def update_bayesian_rating(run_name):
    run = Run.objects.get(name=run_ame)  # Will throw Run.DoesNotExist exceeption
```

Reply:
How to make the `@celery_app.task()` understand to only call it on active runs if we add an argument to it?

Reply:
You can configure explicitly on the admin (periodic task).

![image](https://user-images.githubusercontent.com/13785185/81627346-46619d00-93fe-11ea-9576-909c125c848b.png)

You can configure both positonal and keyword argument.